src/libs/ReportUtils.js

```diff
@@ -1446,7 +1446,6 @@ function buildOptimisticIOUReportAction(type, amount, currency, comment, partici
     // IOUs of type split only exist in group DMs and those don't have an iouReport so we need to delete the IOUReportID key
     if (type === CONST.IOU.REPORT_ACTION_TYPE.SPLIT) {
         delete originalMessage.IOUReportID;
-        originalMessage.participants = [currentUserEmail, ..._.pluck(participants, 'login')];
```

Comment:
Just to double check, we're intentionally not setting this anymore?

Reply:
That's correct. We set `participantAccountIDs` instead on the line below.